pkg/runtime/types.go

```diff
@@ -80,17 +86,19 @@ type TypeMeta struct {
 //
 // +protobuf=true
 type RawExtension struct {
-	// RawJSON is the underlying serialization of this object.
-	RawJSON []byte
+	// Raw is the underlying serialization of this object.
```

Comment:
Thanks for detailed explanation - that helps a lot!

So I guess, we should remove ContentEncoding and ContentType from here, right?
Will do that tomorrow.